dagshub/upload/wrapper.py

```diff
@@ -24,6 +27,57 @@ def get_default_branch(src_url, owner, reponame, auth):
     return res.json().get('default_branch')
 
 
+def create_repo(repo_name, is_org=False, org_name="", description="", private=False, auto_init=False, gitignores="",
+                license="", readme="",
+                template="notebook-template"):
```

Comment:
I do think that the default should be to only have a gitignore, and not a big template with various junk
```suggestion
def create_repo(repo_name, is_org=False, org_name="", description="", private=False, auto_init=False,
                 gitignores="Python", license="", readme="", template=""):
```